IOPool/Tests/test/testDifferentProductRegistriesWrite_cfg.py

```diff
@@ -0,0 +1,57 @@
+
+
+import FWCore.ParameterSet.Config as cms
+import argparse
+from FWCore.Modules.modules import EmptySource
```

Comment:
Could the import be moved next to the use of `EmptySource`?

Reply:
moved